share/completions/tmux.fish

```diff
@@ -20,6 +20,10 @@ end
 #don't allow dirs in the completion list...
 complete -c tmux -x
 
+# Complete even commands not explicitly listed below, as long as `tmux list-commands` works
+set -l all_commands (tmux list-commands -F "#{command_list_name} #{command_list_alias}" 2>/dev/null)
+and complete -c tmux -n __fish_use_subcommand -a "$all_commands"
```

Comment:
Is there a way to ensure this completion is always tried last, only if no manual completion matches?

Putting it first in this file makes it not interfere with anything else *in this file*,
but I worry a bit about any users/distributions/etc that define custom tmux completions (I doubt
there are many, but still).

Reply:
The completions for all the flags are now also provided as a separate commit. If it's harder to review, merging just the first of the commits is an option.

Reply:
> Is there a way to ensure this completion is always tried last, only if no manual completion matches?

After the matrix discussion about rclone [1](https://matrix.to/#/!YLTeaulxSDauOOxBoR:matrix.org/$L4rb8F0WDM3GSw0ZOtJkSbBqZT-QuFH3vWwL6fewqsw?via=matrix.org&via=gitter.im&via=one.ems.host), [2](https://matrix.to/#/!YLTeaulxSDauOOxBoR:matrix.org/$BHA9IgsHKsRXc5WdflpUY3IdbaL-I5EdKppDjuHss1A?via=matrix.org&via=gitter.im&via=one.ems.host), I think this is fine as is. The built-in completions are not meant to be evaluated if the user provides their own earlier in the `fish_complete_path`.

So, I'll resolve this for now.